renderer/configuration/robotouille_config.json

```diff
@@ -73,6 +86,54 @@
                 },
                 "constants": {}
             },
+            "hotdog": {
+                "assets": {
+                    "default": "hotdog.png",
+                    "grilled": {
```

Comment:
To make it so that you do not need the commented portion in canvas.py, make all of these separate dictionaries. so one for grilled, one for grilled_with_ketchup, grilled_with_mustard, grilled_with_ketchup_mustard, and add multiple predicates to the list. So for example, a grilled hotdog with mustard and ketchup would be ["isgrilled", "has_mustard", "has_ketchup"]